client/pages/Assessment/Assessment.scss

```diff
@@ -62,8 +62,8 @@
   grid-area: content;
   height: 100%;
   padding: $spacing-xs;
-  overflow-x: auto;
-  overflow-y: hidden;
+  // overflow-x: auto;
+  // overflow-y: hidden;
```

Comment:
Should we remove unused code comments or is it left for reference?

Reply:
this must be restored